src/StructuredLogger/BinaryLogger/ExtendedPropertyReassignmentEventArgs.cs

```diff
@@ -28,26 +28,8 @@ public ExtendedPropertyReassignmentEventArgs(
             string helpKeyword = null,
             string senderName = null,
             MessageImportance importance = MessageImportance.Low)
-            : base(subcategory: null, code: null, file: file, lineNumber: line, columnNumber: column, 0, 0, message, helpKeyword, senderName, importance)
+            : base(propertyName, previousValue, newValue, location: message, message)

```

Comment:
Is there a reason why we can't just call this constructor directly?  Why do we need our own type?